L/LLVM/common.jl

```diff
@@ -88,8 +88,10 @@ fi
 CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=False)
 CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_HOST_TOOLCHAIN})
 
+(( nproc=1+nproc/3 ))
+
```

Comment:
Note that since you added these lines, the number of parallel jobs here in Yggdrasil was reduced from 64 to 16, so this line can just be removed now
```suggestion
```